### DISCUSSION TITLE
Enabling search may interfere with content

### DISCUSSION THREAD

**Discussion by cagix** [McShelby/hugo-theme-relearn#678] [Q&A] [answered]
Just for information: If you happen to have some content with the name `search` (e.g. `search/index.md` or `search/wuppie/_index.md` ...) and a `search.html` is generated from it, then activating the search functionality will overwrite your content page (landing page) without any warning. 

This should be documented somewhere.

**Comment by McShelby** [comment, marked as answer]
That's also an issue if you have eg. an own `tags` directory or other built-in pathes. You can always run `hugo --printPathWarnings` to get warnings.

**Reply by cagix** [reply]
Yeah, in hindsight it certainly is quite obvious. However, in my case it came a bit as a surprise - I would have appreciated a short statement/warning in the documentation regarding the search feature :)

**Reply by McShelby** [reply]
I will add that to the docs, but I bet that wouldn't stopped you from running into this in the first place ;-)
